commit 797ba18741a03c6a1ccd8070b5f2629bcd6cf87f
Author: orbea <orbea@riseup.net>
Date:   Tue May 11 07:13:35 2021 -0700

    modules/xml: Rename the bool variable.
    
    This fixes compilation failures under some environments.
    
      xml_XML_TreeParser.c: In function ‘ferite_xml_XML_TreeParser_keepBlanks_b’:
      xml_XML_TreeParser.c:10:9: error: two or more data types in declaration specifiers
         10 |    char bool = FE_FALSE;
            |         ^~~~
      xml_XML_TreeParser.c:10:14: error: expected identifier or ‘(’ before ‘=’ token
    
    Signed-off-by: orbea <orbea@riseup.net>

diff --git a/modules/xml/xml_XML_TreeParser.c b/modules/xml/xml_XML_TreeParser.c
index 886a901..25a2105 100644
--- a/modules/xml/xml_XML_TreeParser.c
+++ b/modules/xml/xml_XML_TreeParser.c
@@ -7,17 +7,17 @@
 
 FE_NATIVE_FUNCTION( ferite_xml_XML_TreeParser_keepBlanks_b )
 {
-   char bool = FE_FALSE;
+   char fe_bool = FE_FALSE;
    FeriteObject *self = FE_CONTAINER_TO_OBJECT;
    FeriteObject *super = FE_CONTAINER_TO_OBJECT;
 
-   ferite_get_parameters( params, 1, &bool );
+   ferite_get_parameters( params, 1, &fe_bool );
 
    { /* Main function body. */
 #line 417 "xml.fec"
  
 	XMLDoc *tree = (XMLDoc*)self->odata;
-	tree->keepBlanks = bool;	
+	tree->keepBlanks = fe_bool;
  
    }
    FE_RETURN_VOID;
@@ -151,17 +151,17 @@ FE_NATIVE_FUNCTION( ferite_xml_XML_TreeParser_saveToFile_s )
 
 FE_NATIVE_FUNCTION( ferite_xml_XML_TreeParser_validation_b )
 {
-   char bool = FE_FALSE;
+   char fe_bool = FE_FALSE;
    FeriteObject *self = FE_CONTAINER_TO_OBJECT;
    FeriteObject *super = FE_CONTAINER_TO_OBJECT;
 
-   ferite_get_parameters( params, 1, &bool );
+   ferite_get_parameters( params, 1, &fe_bool );
 
    { /* Main function body. */
 #line 403 "xml.fec"
 
  
- if( bool == FE_FALSE )
+ if( fe_bool == FE_FALSE )
  xmlDoValidityCheckingDefaultValue = 0;
  else
  xmlDoValidityCheckingDefaultValue = 1;
